django/cantusdb_project/main_app/templatetags/helper_tags.py

```diff
@@ -276,3 +277,12 @@ def join_absolute_url_links(
         '<b><a href="{0}">{1}</a></b>',
         ((obj.get_absolute_url(), getattr(obj, display_attr)) for obj in objects),
     )
+
+
+@register.filter
+def render_markdown(value: str) -> SafeString:
+    """
+    Renders markdown text as HTML.
+    """
+    html: str = github_flavored_markdown_to_html(value)
+    return mark_safe(html)
```

Comment:
Realizing it would be useful if this had a comment...
```suggestion
    # Generated html is marked safe b/c cmark is run in safe mode in
    # github_flavored_markdown_to_html
    return mark_safe(html)
```